src/pyvesync/vesync.py

```diff
@@ -21,7 +21,7 @@
 
 
 class VeSync(object):
-    def __init__(self, username, password, time_zone):
+    def __init__(self, username, password, time_zone=None):
```

Comment:
Does regex.search raise an error if None is passed?  Would time_zone=DEFAULT_TZ be better?

Reply:
the regex.search block wouldn't be hit in the case of None since it would fail the isinstance check and go into the else block where it's set to DEFAULT_TZ. I've got some tests for this in another branch that I could use some help with.

Reply:
I've updated the code to set time_zone=DEFAULT_TZ as you suggested as well

Reply:
😬, the isinstance check line was hidden . Is it ok to merge?

Reply:
yup